src/openssl/ssl/socket.cr

```diff
@@ -53,15 +53,13 @@ abstract class OpenSSL::SSL::Socket < IO
   class Server < Socket
     def initialize(io, context : Context::Server = Context::Server.new, sync_close : Bool = false)
       super(io, context, sync_close)
-      begin
-        ret = LibSSL.ssl_accept(@ssl)
-        unless ret == 1
-          io.close if sync_close
-          raise OpenSSL::SSL::Error.new(@ssl, ret, "SSL_accept")
-        end
-      rescue ex
-        LibSSL.ssl_free(@ssl) # GC never calls finalize, avoid mem leak
-        raise ex
+    end
+
+    def accept
```

Comment:
Isn't this a breaking change?

Reply:
That's true. I just refactored this fix so it doesn't break compatibility. Thanks!